### PR TITLE
6869 chart order population

### PR DESCRIPTION
**Pull request by stephensprinkle**

### CODE REVIEW

**Comment by stephensprinkle** [comment]
To note — there are some errors present related to the charts — I’ll be taking care of those as part of: https://app.clubhouse.io/augur/story/4686/market-charts-fine-tune-styling-functionality-pass

File: src/modules/market/components/market-outcome-charts--depth/market-outcome-charts--depth.jsx [hunk 1]
```diff
@@ -179,12 +184,25 @@ export default class MarketOutcomeDepth extends Component {
...
 
           // Determine closest order
-          const hoveredPrice = this.state.yScale.invert(mouse[1])
+          const hoveredPrice = this.state.yScale.invert(mouse[1]).toFixed(this.props.fixedPrecision)
 
           this.props.updateHoveredPrice(hoveredPrice)
         })
         .on('click', () => {
-          console.log('TODO -- fill order form')
+          const mouse = d3.mouse(d3.select('#outcome_depth').node())
```
Comment (justinbarry) [inline src/modules/market/components/market-outcome-charts--depth/market-outcome-charts--depth.jsx]: Is this "scoped" to the current component somehow?

Comment (stephensprinkle) [reply src/modules/market/components/market-outcome-charts--depth/market-outcome-charts--depth.jsx]: Which aspect are you referencing exactly?

_Guessing_ you’re referencing the `select`?  

If so, that is querying the DOM globally; thus, the `#`; however, it only matches the first result.

Comment (stephensprinkle) [reply src/modules/market/components/market-outcome-charts--depth/market-outcome-charts--depth.jsx]: Might make sense to make that ID more specific?

Comment (justinbarry) [reply src/modules/market/components/market-outcome-charts--depth/market-outcome-charts--depth.jsx]: I meant the selector. Will we ever have 2 of these up at once? If so, will that cause a problem?

Comment (stephensprinkle) [reply src/modules/market/components/market-outcome-charts--depth/market-outcome-charts--depth.jsx]: Re: 2 at the same time, no for the depth chart (at least per the mocks)

Comment (justinbarry) [reply src/modules/market/components/market-outcome-charts--depth/market-outcome-charts--depth.jsx]: Cool.

**Comment by stephensprinkle** [comment]
:sparkles:

**Pull request merged**